profiles/singlecell-mongo-sa/terraform/singlecell-mongo-sa/sa.tf

```diff
@@ -0,0 +1,11 @@
+resource "google_service_account" "app" {
+  account_id   = "${var.owner}-${var.service}"
```

Comment:
it would be better for the path to this file to be `profiles/singlecell-mongo-sa/terraform/singlecell-mongo/sa.tf` rather than `profiles/singlecell-mongo-sa/terraform/singlecell-mongo-sa/sa.tf` . The service name comes from the most-specific directory name and the service is `singlecell-mongo` not `singlecell-mongo-sa`

Reply:
Done in 84e2f6b.